src/__test__/Home.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import store from '../redux/store/index';
+import Home from '../Components/Home';
+
+describe('Missions', () => {
+  it('renders Missions component', () => {
+    const m = renderer.create(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Home />
+        </BrowserRouter>
+      </Provider>,
+    ).toJSON();
+    expect(m).toMatchSnapshot();
+  });
```

Comment:
- [ ] Kindly write integration tests for your components using **_React Testing Library_** based on the requirement. **_React testing Library_** is used to test how an end user will interact with your application. You can read this [article](https://www.robinwieruch.de/react-testing-library/) for guidance 👍 😎